index.js

```diff
@@ -0,0 +1,55 @@
+var Random = function() {
+}
+var chooseRandomNumer = function(startindex, endIndex){
+	return Math.floor(Math.random() * (startindex - endIndex)) + endIndex;
+}
+// Can we use an object instead of startindex and max index???
+Random.prototype.getRandomNumberInRangeSync = function(array, number, startIndex, endIndex) {
+    if(startIndex < 0 || endIndex > array.length) {
+//return error
```

Comment:
Lets break up this if clause and have specific errors like 
`throw new Error('Invalid startIndex value");` 
`throw new Error('Invalid endIndex value");`